### ISSUE TITLE
Build on Linux

### ISSUE THREAD

**Issue by christiana** [IGSIO/OpenIGTLinkIO#1]
The current (first) version is tested on MacOSX 10.10 only.  Should be tested for Linux.

**Comment by drouin-simon** [comment]
Just built on Ubuntu 14.04, using the CTK and VTK from the Slicer superbuild. I have a bunch of warnings all due to some VTK header file. I haven't tested anything yet.

**Comment by drouin-simon** [comment]
The example crashes on Linux. It seems to be related to the fact that I build with Qt5 but use Slicer's build of VTK and CTK, which is Qt4. I can easily build VTK with Qt5, but CTK is a different story. Does it even work with Qt5? Christian, how to you do it?

**Comment by lassoan** [comment]
CTK works with Qt5 (MITK uses CTK with Qt5)

**Comment by drouin-simon** [comment]
Ok thanks. Just realized that trying to build CTK independently from Slicer. I was confused by the CTK web site that emphasize Qt5 support is experimental.

**Comment by christiana** [comment]
CustusX also used CTK with QT5.

**Comment by drouin-simon** [comment]
It is running:
![igtl-client-on-ubuntu](https://cloud.githubusercontent.com/assets/5595829/12761489/32fa842a-c9b9-11e5-88cf-90603966c66c.png)

**Comment by adamrankin** [comment]
Nice!

**Comment by drouin-simon** [comment]
Here is how to do it on Ubuntu 14.04:
1. Build vtk with qt5 (I used vtk 6.3)
2. Build ctk with qt5 and specifying VTK_DIR to point to the one you built (don't let CTK build its own, it will built vtk with qt4). I used default config but enabled these:
   *_CTK_ENABLE_Widgets
   *_CTK_LIB_Widgets
   *_CTK_LIB_Visualization/VTK/Core
   and disabled:
   *_BUILD_TESTING
3. Build openigtlinkIO with the VTK_DIR pointing to vtk built in 1. I used the version of OpenIgtLink that is built in the Slicer superbuild.
